bin/runtime/src/lib.rs

```diff
@@ -549,7 +578,7 @@ parameter_types! {
 pub struct TreasuryGovernance;
 impl SortedMembers<AccountId> for TreasuryGovernance {
     fn sorted_members() -> Vec<AccountId> {
-        vec![pallet_sudo::Pallet::<Runtime>::key()]
+        pallet_sudo::Pallet::<Runtime>::key().into_iter().collect()
```

Comment:
this looks veeery suspicious - `key()` will return `AccountId32` which is actually `[u8; 32]`, so this function does not seem to return collection of accounts, but rather a collection of bytes

correct me if I am wrong

Reply:
`key()` now returns an `Option<AccountID>`, so `into_iter` won't get "inside" the `[u8; 32]` array.

The return type is `Vec<AccountId>` so I don't think it would even compile if it produced collection of bytes

Reply:
oh, I didn't know that trick of converting option to iterator
smart